src/sedcli_main.c

```diff
@@ -67,6 +67,7 @@ static cli_option activatelsp_opts[] = {
 static cli_option reverttper_opts[] = {
 	{'d', "device", "Device node e.g. /dev/nvme0n1", 1, "DEVICE", CLI_OPTION_REQUIRED},
 	{'i', "psid", "Revert Trusted Peripheral (TPer) with the PSID authority", 0, "FLAG", CLI_OPTION_OPTIONAL_ARG},
+	{'n', "non-destructive", "Non-Destructive Revert Trusted Peripheral (TPer) with the PSID authority", 0, "FLAG", CLI_OPTION_OPTIONAL_ARG},
```

Comment:
Can you please change the description to:
Perform non-destructive revert on TPer (i.e. keep the user data intact even after revert)